### ISSUE TITLE
why decay_steps not related with num_clones 

### ISSUE THREAD

**Issue by msunming** [tensorflow/models#4065]
In models/research/slim/train_image_classifier.py, 
i think  decay_steps should = 
     int(num_samples_per_epoch / (FLAGS.batch_size * FLAGS.num_clones )*FLAGS.num_epochs_per_decay);

right?

**Comment by karmel** [comment]
That seems plausible, given that decay steps is typically calculated over global batch size. That said, the use of queue runners have been deprecated, and I would therefore encourage you to take a look at the similar example of calculating learning rate with multiple workers in the [Official Resnet](https://github.com/tensorflow/models/blob/master/official/resnet/imagenet_main.py#L285). If that does not meet your needs, please feel free to submit a PR with data that validates the correctness of the new calculation.
